neural_compressor/adaptor/ox_utils/calibration.py

```diff
@@ -137,7 +139,12 @@ def augment_graph(self, activation_only=False, weight_only=False):
                              (node.name in self.white_nodes)
             if should_be_dump:
                 if not weight_only and not activation_only:
-                    tensors_to_dump.update(node.input)
```

Comment:
'not weight_only and not activation_only' means both weight and activation should be dump, so initializer can't be skipped. Please try other ways to skip initializer.